src/HomePage/NavBar/NavBar.scss

```diff
@@ -10,22 +10,28 @@
 }
 
 .navbar {
-  margin: 10px;
+  margin: 5px auto;
   display: flex;
-  justify-content: space-between;
-}
+  flex-direction: row;
+  font-family: $title-font-family;
+  font-size: 16px;
+  width: 500px;
+  justify-content: space-around;
+
+  @media (max-width: 500px) {
+    width: 450px;
```

Comment:
Whats the reasoning for having this? Doesn't really affect the mobile friendliness of this navbar that much

Reply:
Main reason for this was to prevent a side scroll bar from appearing on a small screen, as the navbar would push it out slightly

Reply:
Ok, i would change this to just deal with small screens properly but i dont think you should do this now. Better to fix all the mobile stuff seperately later on